api/DOMQuad.json

```diff
@@ -29,10 +29,10 @@
             "version_added": "45"
           },
           "safari": {
-            "version_added": true
+            "version_added": "11"
```

Comment:
Based upon the linked changeset and its presence in WebKit 603.1.30, shouldn't this be Safari 10.1 (WebKit 603.2.1)?  (This applies to the entire file.)

Reply:
Nope. My linked changeset was the wrong one. Sorry about that. See above for the right one.